src/main/java/data/plans/PlanList.java

```diff
@@ -226,6 +228,26 @@ public void listAllPlan() {
         }
     }
 
+    /**
+     * Prints the plan details, workouts in the plan.
+     */
+    public void listPlanDetails(String userArgument, UI ui) throws NumberFormatException {
+        int indexOfPlan = Integer.parseInt(userArgument.trim());
+        Plan planToViewDetails = getPlanFromIndexNum(indexOfPlan);
+        String planName = getPlansDisplayList().get(indexOfPlan - 1);
+
+        ArrayList<Workout> workoutsInPlanList = planToViewDetails.getWorkoutsInPlanList();
+        int numberOfWorkoutsInPlan = workoutsInPlanList.size();
+        assert (numberOfWorkoutsInPlan <= MAX_NUMBER_OF_WORKOUTS_IN_A_PLAN && numberOfWorkoutsInPlan > 0);
```

Comment:
Perhaps each condition can be enclosed in its own parentheses? That is, `assert ((numberOfWorkoutsInPlan <= MAX_NUMBER_OF_WORKOUTS_IN_A_PLAN) && (numberOfWorkoutsInPlan > 0));`

If you want, perhaps you could also add a message for when the assertion fails.